cmd/concourse/concourse_suite_test.go

```diff
@@ -11,3 +12,19 @@ func TestConcourse(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "Concourse Suite")
 }
+
+var concoursePath string
+
+var _ = SynchronizedBeforeSuite(func() []byte {
```

Comment:
Ah, makes sense - the red builds had me worried but compiling it on a single node should be solid (we build the binary elsewhere in our pipeline already).